plugins/foreman_discovery/nightly/index.md

```diff
@@ -367,6 +366,14 @@ change the default entry from localboot entry to Discovery:
 
     ONTIMEOUT discovery
 
+Similar change is needed in *PXEGrub2 global default* template:
+
+    default=discovery
+
+The same way *PXEGrub1 global default* must be changed, but this is a legacy
```

Comment:
This doesn't read well to me. Perhaps `*PXEGrub1 global default* must be changed in the same way,` would be better?